### PR TITLE
Integrated Past Appointments

### PR DESCRIPTION
**Pull request by garzeah**
### What this PR does (required):
- Updated our endpoint to return past appointments
- Wrote logic on the front-end to fetch past appointments
- Code was written in a way where it will be easy to add "All", "Future", etc. types of appointments

### Screenshots / Videos (required):
Screen Shot 2021-07-21 at 12 38 21 PM

### Any information needed to test this feature (required):
- Login and once you are in the dashboard click "Scheduled Events"

### CODE REVIEW

File: client/src/helpers/APICalls/appointment.ts [hunk 1]
```diff
@@ -1,6 +1,15 @@
 import { RouteComponentProps } from 'react-router-dom';
 import { appointmentInfoProp, appointmentProp } from '../../interface/AppointmentProps';
 
+const fetchAppointments = async (username: string, type: string): Promise<void> => {
+  const test = await fetch(`/appointment?username=${username}&type=${type}`, {
+    method: 'GET',
+    credentials: 'include',
+  });
+
+  console.log(test);
```
Comment (ajaytomgeorge) [inline client/src/helpers/APICalls/appointment.ts]: Please remove the console.log

File: server/controllers/appointment.js [hunk 2]
```diff
@@ -16,7 +17,32 @@ exports.fetchAppointments = asyncHandler(async (req, res, next) => {
...
+      currentTime = new Date(
+        convertToTimeZone(currentTime, appointments[key].timezone)
+      );
+
+      // If the appointment time is before current time, then push into pastAppointments
+      if (currentTime.getTime() < appointmentTime.getTime())
+        pastAppointments.push(appointments[key]);
+    }
+
+    res.status(200).json({ success: { appointments: appointments } });
```
Comment (ajaytomgeorge) [inline server/controllers/appointment.js]: Aren't supposed to return pastAppointments? :-)

Comment (garzeah) [reply server/controllers/appointment.js]: Ah wow haha, yeah I just changed it! Thank you!

**Review by moffatethan — APPROVED** [review]
Good to go Andy!

**Pull request merged**